deeprankcore/utils/exporters.py

```diff
@@ -253,13 +253,11 @@ def __enter__(self):
 
     def __exit__(self, exception_type, exception, traceback):
 
-        if self.phase == 'validation':
-            self.phase = 'training'
-
-        self.df.to_hdf(
-            os.path.join(self._directory_path, 'output_exporter.hdf5'),
-            key=self.phase,
-            mode='a')
+        if self.phase is not None:
```

Comment:
When the training phase is over, and the exit method is called, we want to save the outputs with the 'training' key. Validation is part of the training in this regard. I would leave the code as it was here.

Reply:
I originally made this change to fix nontype errors. In 64cb8eb89df0c73cc3abeb9d20483c42da24603c I made sure that validation is converted to training.